action/protocol/actpool.go

```diff
@@ -0,0 +1,31 @@
+package protocol
+
+import (
+	"sync"
+
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/pkg/errors"
+
+	"github.com/iotexproject/iotex-core/action"
+)
+
+// ActPoolInterface is the interface for actpool
+type ActPoolInterface struct {
```

Comment:
Why is `ActPoolInterface` necessary? Can we just use ActPoolManager interface instead?